test/CMakeLists.txt

```diff
@@ -69,11 +69,9 @@ endif()
 add_custom_target(build-onedpl-tests
     COMMENT "Build all oneDPL tests")
 
-if (ONEDPL_USE_DR)
-    set(run-onedpl-tests-depends build-onedpl-tests shp-tests shp-tests-3)
-else()
-    set(run-onedpl-tests-depends build-onedpl-tests)
-endif()
+set(run-onedpl-tests-depends build-onedpl-tests)
```

Comment:
why is this change?

Why builing shp-tests* was removed if ONEDPL_USE_DR is set (which means there is right C++ standard to run DR code)?

Reply:
my fault probably, I thought that C++23 is not supported and we only compile and run tests of our code in a docker container